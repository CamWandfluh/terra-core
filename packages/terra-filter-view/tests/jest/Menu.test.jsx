import React from 'react';
import Menu from '../../src/_Menu';
import intlContexts from './intl-context-setup';

describe('Menu', () => {
  it('should render a dropdown Menu', () => {
    const menu = <Menu onSelect={() => {}} value="value" />;
    const wrapper = shallow(menu, intlContexts.shallowContext);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render a Menu with no results', () => {
    const menu = <Menu onSelect={() => {}} value="value" />;
    const wrapper = shallow(menu, intlContexts.shallowContext);
    expect(wrapper).toMatchSnapshot();
  });
});
