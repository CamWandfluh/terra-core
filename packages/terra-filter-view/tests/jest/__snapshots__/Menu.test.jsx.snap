// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render a Menu with no results 1`] = `
<ul
  className="menu"
  role="listbox"
>
  <NoResults
    key=".0"
  />
</ul>
`;

exports[`Menu should render a dropdown Menu 1`] = `
<ul
  className="menu"
  role="listbox"
>
  <NoResults
    key=".0"
  />
</ul>
`;
